saleor/payment/error_codes.py

```diff
@@ -16,3 +16,4 @@ class PaymentErrorCode(Enum):
     NOT_SUPPORTED_GATEWAY = "not_supported_gateway"
     CHANNEL_INACTIVE = "channel_inactive"
     BALANCE_CHECK_ERROR = "balance_check_error"
+    EMAIL_NOT_SET = "email_not_set"
```

Comment:
Maybe we should use `CHECKOUT_EMAIL_NOT_SET`. It's could be confusing when Payment mutation responds that "Email not set" and someone thinks that they should set this email on payment object instead of checkout.